sdk/feature-management/test/featureEvaluation.test.ts

```diff
@@ -0,0 +1,242 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as chai from "chai";
+const expect = chai.expect;
+
+import { FeatureManager, ConfigurationObjectFeatureFlagProvider, EvaluationResult, VariantAssignmentReason } from "../";
```

Comment:
I followed the pattern used by @Eskibear https://github.com/microsoft/FeatureManagement-JavaScript/blob/main/test/featureManager.test.ts

Reply:
But I found that the testsuite in JS provider, put all import statements in the beginning. I will make them consistent. But personally, I like the way in this repo. 

Recall that we've had this discussion before: https://github.com/microsoft/FeatureManagement-JavaScript/pull/24#discussion_r1722739750

Reply:
I prefer the imports together. In the /example code- it might be helpful to keep them apart to clearly show the reader which parts relate to what. But I don't think it's valuable outside of an example